src/Azure.ApiHub.Sdk.csproj

```diff
@@ -1,133 +1,36 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+<Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{F8356D27-805D-4ECD-95BF-8CC08063EB02}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>Microsoft.Azure.ApiHub.Sdk</RootNamespace>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
We need a common prop/build properties file that is used by the project in this solution with the expected values (which will also end up in the package metadata). Please see an example here: https://github.com/Azure/azure-webjobs-sdk-extensions/blob/3cfc5d2ce5e7f980861530dab71a9954485e7e69/build/common.props

And this is how this file is used: https://github.com/Azure/azure-webjobs-sdk-extensions/blob/3cfc5d2ce5e7f980861530dab71a9954485e7e69/src/WebJobs.Extensions.Http/WebJobs.Extensions.Http.csproj#L2

Please also set the project specific properties (mainly for packaging) as it is done here: https://github.com/Azure/azure-webjobs-sdk-extensions/blob/3cfc5d2ce5e7f980861530dab71a9954485e7e69/src/WebJobs.Extensions.Http/WebJobs.Extensions.Http.csproj#L3-L8

Reply:
Changes are made in the last commit.